### ISSUE TITLE
DnsServer not a valid member

### ISSUE THREAD

**Issue by awaemmanuel** [PowerShell/xDnsServer#54]
Using this on Azure and I get this error. 

```
+ DnsServer = $dnsServerFqdn
 + ~~~~~~~~~ The member 'DnsServer' is not valid. Valid members are
 'DependsOn', 'Ensure', 'Name', 'PsDscRunAsCredential', 'Target', 'Type', 'Zone'.
```

**Comment by awaemmanuel** [comment]
The latest version 1.8.0 doesn't have this member. The latest github code has this property added. When will it be made available please, or what is a workaround to mitigate this?

**Comment by johlju** [comment]
Closing this as per previous comment, that this is resolved in v1.8.0.0 an that was release some time ago (we are now up to v.1.10.0.0). If this was closed in error, then please reopen or comment.
